compiler/il/OMRResolvedMethodSymbol.hpp

```diff
@@ -303,6 +303,22 @@ class OMR_EXTENSIBLE ResolvedMethodSymbol : public TR::MethodSymbol
    void clearProfilingOffsetInfo();
    void dumpProfilingOffsetInfo(TR::Compilation *comp);
 
+   /**
+    * @brief Detects whether a parameter is variant.
+    *
+    * A variant parameter is one that is written to, potentially with a
```

Comment:
The type of the parm symbol doesn't change, it's the value it refers to that may change. For address type, a variant parameter may refer to different objects potentially objects of different classes, which makes the hint (the parameter signature) untrusted.

Reply:
Thanks for clarifying.

Reply:
Fixed in force push.